pkg/cmd/pr/create/create.go

```diff
@@ -266,14 +266,14 @@ func NewCmdCreate(f *cmdutil.Factory, runF func(*CreateOptions) error) *cobra.Co
 func createRun(opts *CreateOptions) (err error) {
```

Comment:
Should the named return be removed, too?

```suggestion
func createRun(opts *CreateOptions) error {
```

Reply:
Yeah, that change got stranded on my local machine. I've put it into the correct commit, now. Thanks!